lib/public_api.ts

```diff
@@ -0,0 +1,11 @@
+export * from './src/app-switcher/app-switcher.module';
```

Comment:
Is there any need to also include other exports besides modules? The scenario I'm thinking of is if somebody uses a cashmere component in their app and needs to declare the cashmere component in their test bed. Perhaps the module itself makes this available but I wasn't sure.

Reply:
so, I was just following the advice from the Packaging Angular talk to only export modules.  I did notice that ng-packagr (via ngc) does export individual components, but that shouldn't be something that consumers of the package should normally have to worry about.